music/src/core/player.ts

```diff
@@ -47,9 +47,13 @@ function compareQuantizedNotes(a: NoteSequence.INote, b: NoteSequence.INote) {
  * played.
  */
 export abstract class BasePlayerCallback {
-  /* Will be called for each time/note pair in a sequence being played.
+  /**
+   *   Will be called for each time/note pair in a sequence being played.
```

Comment:
Sorry to nit, but there should just be a single space after the *.

Reply:
nit away! done.
i was basing myself on the spacing used in the BasePlayer constructor docstring.